nyxx_lavalink/lib/src/model/GuildPlayer.dart

```diff
@@ -0,0 +1,39 @@
+part of nyxx_lavalink;
+
+/// A player of a specific guild
+class GuildPlayer {
+  /// Track queue
+  List<QueuedTrack> queue = [];
+  /// The currently playing track
+  QueuedTrack? nowPlaying;
```

Comment:
these are still user writeable, nyxx isnt consisten about that, but we should prevent that where possible. Should be probablu marked as final

Reply:
i want them to be writeable, if the user forces a track to play, it's intended to not set as `nowPlaying`, and the user has to be able to remove elements from queue